VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/config/E2EConfig.java

```diff
@@ -0,0 +1,6 @@
+package org.edx.mobile.test.feature.config;
+
+public class E2EConfig {
+    public static String EMAIL_TEMPLATE = "test-{unique_id}@example.com";
```

Comment:
hmmm, if by config you mean the ones we have in `Config.java` file which is populated by the sandbox config repo, then I did check them and didn't find any block for E2E configs.

Am I missing something (cuz I looked at the iOS code and have somehow matched it) ?